pages/categories/operational.md

```diff
@@ -41,10 +41,63 @@ As the organisation does not own or control the IT assets delivering these servi
+
+| Provider                      | PUE        |
+| ----------------------------- | ---------- |
+| Alibaba Cloud                 | 1.3        |
+| AWS                           | 1.135      |
+| Azure                         | 1.185      |
+| GCP                           | 1.1        |
+| Oracle Cloud                  | 1.15       |
+| Worldwide Data centre Average | 1.58 - 1.8 |
+
+Sources: [^ccf-pue] [^oracle-pue]
+
+[^ccf-pue]: [Cloud Carbon Footprint Methodology](https://www.cloudcarbonfootprint.org/docs/methodology#appendix-i-energy-coefficients)
+[^oracle-pue]: [Oracle Climate Action](https://www.oracle.com/uk/government/climate-action/#rc30p2)
+
+These figures should be considered carefully when sourced from the Cloud Providers themselves, as it may apply to specific data centres only rather than being an average. In future it is hoped that these figures can be further reduced by recapturing the waste heat from data centres, with [Deep Green](https://deepgreen.energy/) being an example of a smaller Cloud provider whose micro data centres extract heat to provide hot water for a range of purposes.
+
+The region that the services are run from can also have a significant impact on carbon emissions, depending on the mix of renewable energy sources that power the underlying data centres. Some cloud providers may report that they use 'zero carbon' energy but it's important to distinguish between direct renewable power to the data centre and that which has been offset via [Power Purchase Agreements](/glossary#power-purchase-agreements-ppa) or [Renewable Energy Certificates](/glossary#renewable-energy-certificates-recs). The flexibility of the cloud does at least make it easier to switch location than it would be to move an on-premise estate.
+
+Returning to the IaaS and PaaS definitions, these can also be viewed through an emissions lens. IaaS may allow the most control over cloud resources but it retains the same issues that can be seen when running on-premise, with instances potentially being unused for the majority of time. PaaS may provide more opportunities for the Cloud provider to choose more efficient usage patterns behind the scenes. This extends to so called 'Serverless' options, which provide ways of allocating resources that only exist when work is happening, minimising the impact of idle hardware. Unfortunately the term has become slightly overloaded, with some services still requiring a minimum sized instance to run at all times, so careful reading of documentation may be necessary to determine if services can truly scale to zero.
+
+Cloud services rely on virtualisation of the hardware to utilise it most effectively, allowing for a higher average utilisation than most on-premise data centres. This can be in the range of ~50% utilisation compared to ~20% in a traditional data centre.[^cloud-utilisation] Cloud providers offer 'Spot Instances' as a way of making use of hardware that would otherwise be underutilised.
```

Comment:
```suggestion
Cloud services rely on virtualisation to share their server hardware between multiple tenants, allowing for a higher average utilisation than most on-premise data centres. This can be in the range of ~50% utilisation compared to ~20% in a traditional data centre.[^cloud-utilisation] Cloud providers also offer 'Spot Instances' as a way of making use of hardware that would otherwise be underutilised.
```